index.d.ts

```diff
@@ -4,8 +4,8 @@ import { Operator } from 'rxjs/Operator';
 
 export declare class ActionsObservable extends Observable<Action> {
   constructor(actionsSubject: Observable<Action>);
-  lift(operator: Operator<any, Action>);
-  ofType(...key: any[]);
+  lift(operator: Operator<any, Action>) : ActionsObservable;
+  ofType(...key: any[]) : boolean;
```

Comment:
I think that lift should explicitly return ActionsObservable but I'm not sure about ofType.
(full disclosure: I'm still trying to grok RxJS.)

ofType does a filter thing, which returns an Operator, but Operator is Observable, right? So... explicitly returning Operator doesn't seem wrong but I'm not sure that's semantically correct.